foreign_cc/meson.bzl

```diff
@@ -208,8 +208,8 @@ def meson_with_requirements(name, requirements, **kwargs):
 
     meson_tool(
         name = "meson_tool_for_{}".format(name),
-        main = "@meson_src//:meson.py",
-        data = ["@meson_src//:runtime"],
+        main = "@meson_1.1.1_src//:meson.py",
+        data = ["@meson_1.1.1_src//:runtime"],
```

Comment:
@jheaff1 any ideas how I could avoid this hardcoded version number?
I want the end user to not have to do use_repo(tools, "meson_src") if they're using this macro which they currently have to; maybe we should make the toolchain variants have to be declared via a module extension and then generate this toolchain into the toolchain_hub e.g.
```
tools.meson(version = "1.1.1")
tools.meson(version = "1.1.1", requirements = ["@pypi__numpy"])
```
?